itest/rust/src/register_tests/func_test.rs

```diff
@@ -89,6 +111,35 @@ impl GdSelfReference {
         f1 && f2
     }
 
+    #[func]
+    fn cfg_removes_function() -> bool {
+        true
+    }
+
+    #[cfg(any())]
+    #[func]
+    fn cfg_removes_function() -> bool {
+        false
+    }
+
+    #[func]
+    #[cfg(any())]
+    fn cfg_removes_function() -> bool {
+        false
+    }
+
+    #[cfg(any())]
+    #[func]
+    fn cfg_removes_function_ffi_glue() -> bool {
+        true
+    }
+
+    #[func]
+    #[cfg(any())]
+    fn cfg_removes_function_ffi_glue() -> bool {
+        true
+    }
```

Comment:
Also here, not exactly clear what "FFI glue" means 🤔

Reply:
I've renamed the tests now, should probably be clearer